obspy/clients/eida/fetch.py

```diff
@@ -0,0 +1,735 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+################################################################################
+# (C) 2014-2016 Helmholtz-Zentrum Potsdam - Deutsches GeoForschungsZentrum GFZ #
+#                                                                              #
+# License: LGPLv3 (https://www.gnu.org/copyleft/lesser.html)                   #
+################################################################################
+
+"""
+A command-line FDSN Web Service client with EIDA routing and authentication support.
+"""
+import sys
+import time
+import datetime
+import optparse
+import threading
+import socket
+import csv
+import re
+
+try:
+    # Python 3.2 and earlier
+    from xml.etree import cElementTree as ET
```

Comment:
We don't support Python 3.2.. does it even run on 3.2? Is there a strong need for this one? We wouldn't notice if it breaks in the future..